centreon/www/front_src/src/Dashboards/SingleInstancePage/Dashboard/AddEditWidget/WidgetProperties/Inputs/ResourcesTableDisplayType/icons/getIconByView.ts

```diff
@@ -0,0 +1,16 @@
+import { equals } from 'ramda';
+
+import viewByService from './view_service.svg';
+import viewByHost from './view_host.svg';
+import viewByAll from './view_all.svg';
```

Comment:
The svg must be inline svg instead of file svg such as Mui icon. https://mui.com/material-ui/icons/#svgicon
But the icon must have a viewport of "0 0 24 24"

Reply:
It reduces the time to display because it doesn't download the svg file